README.md

```diff
@@ -1 +1,47 @@
-# Phantom-Backend-Pirates
\ No newline at end of file
+# Phantom-Backend-Pirates
+
+
+# node-sequelize-postgres-crud
+
+Basic CRUD operation by using node.js and typeORM.
+
+## Installation
+
+- You need to create database in `postgres` with `YOUR_DATABASE_NAME`
+- `git clone https://github.com/atlp-rwanda/Phantom-Backend-Pirates.git`
+- `cd Phantom-Backend-Pirates`
+- `npm install`
+
+- Create .env file on root folder
+- `touch .env`
+
+- Enter line
+LOCAL_PORT = your port 
+- edit DB_URL basing on your database configuration
+DB_URL = postgres://<username>:<your_password>@localhost:5432/<your_database>?sslmode=disable
+
+- Go in `src/config/config.js` based on your database
+
+
```

Comment:
create a proper folder structure that makes it easy to add dependencies for postgres and others.

Reply:
ok lets look at it